lib/uniris/application.ex

```diff
@@ -43,7 +45,12 @@ defmodule Uniris.Application do
     ]
 
     opts = [strategy: :rest_for_one, name: Uniris.Supervisor]
-    Supervisor.start_link(Utils.configurable_children(children), opts)
+    res = Supervisor.start_link(Utils.configurable_children(children), opts)
+
+    :telemetry.execute([:uniris, :run_app, :success], %{"latency" => 1.2}, %{"hello" => "uniris"})
```

Comment:
I think those measurements were here for tests . Maybe a cleanup ..

Reply:
You mentioned you want a simple example of telemetry usage. That's it.

Reply:
I wanted a simple setup but ok will change later

Reply:
Let me clean these rows. We have a setup and it works as expected.